core/main/src/Core/Container.ts

```diff
@@ -393,4 +392,13 @@ export class Container {
 
         this.density = (canvas.width * canvas.height) / (densityOptions.factor * pxRatio * densityOptions.area);
     }
+
+    private loadTheme(name?: string): void {
+        if(name){
```

Comment:
This if needs an else searching for a default theme

Reply:
But default name string is empty.

Reply:
yes, but there's a default property on Theme class, you have to search for the one with true value